js/core/dom_component.js

```diff
@@ -12,7 +12,7 @@ import { getPublicElement } from '../core/element';
 import { grep, noop } from './utils/common';
 import { inArray } from './utils/array';
 import { isString, isDefined } from './utils/type';
-import { hasWindow } from '../core/utils/window';
+import windowUtils from '../core/utils/window';
```

Comment:
Why is `{ hasWindow } ` changed to windowUtils?

Reply:
Fixed